---
# This page uses Hydejack's `about` layout, which shows the primary author's picture and about text at the top.
# You can change it to the regular `page` layout if you want.
layout: page

# The title of the page.
title: Resume/CV

# Write a short (~150 characters) description of each blog post.
# This description is used to preview the page on search engines, social media, etc.
description: >
  Lorem ipsum dolor sit amet, consectetur adipiscing elit,
  sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.

# You can show the description on the page by deleting this line:
hide_description: true

# TODO
# image: /assets/img/blog/hydejack-8.png
---

[PDF](../assets/Resume.pdf)

[Email](mailto:mirudhula.tm@gmail.com) |
<a href="https://www.linkedin.com/in/mirudhula/" target="_blank">Linkedin</a> |
<a href="https://mirudhula-m.github.io" target="_blank">Website</a> |
<a href="https://github.com/Mirudhula-m" target="_blank">Github</a>




## **Education** 

<span style="color:gray">September 2021 - May 2023</span><br>
**Carnegie Mellon University**, *Pittsburgh, USA*
<br>School of Computer Science
<br>Master of Science in Computational Biology

<span style="color:gray">August 2016 - May 2020</span><br>
**PES University**, *Bangalore, India*
<br>Bachelor of Technology in Biotechnology (Major)
<br>Bachelor of Technology in Computer Science (Minor)


## **Skills**

**Programming Languages**
<br>Python, Go, C, Java, R, MATLAB

**Frameworks**
<br>Pytorch, Tensorflow, Keras, Skorch, Scikit-learn

**Web Development**
<br>HTML, Javascript, CSS

**Database Management**
<br>PostgreSQL

**Tools**
<br>Git, AWS Cloud



## **Experience**

<span style="color:gray">January 2022 - Present</span><br>
**Research Assistant**<br>
*Lee Lab, Carnegie Mellon University, Pittsburgh, USA*

* Analyzing macaque EEG data to evaluate evolution of neural firingrates, sparsity and noise correlations.
* Identifying the application of Community Structures on the data.
* Utilizing predictive coding to model spatiotemporal patterns in the brain.


<span style="color:gray">June 2020 - July 2021</span><br>
**Project Assistant**<br>
*Cognition Lab, Indian Institute of Science, Bangalore, India*

* Designed and developed 2-ADC behavior tasks for investigating the effect of cognitive load on attention.
* Ran initial cohort of subjects to analyze and quantify behavior.
* This is still a work-in-progress project at the lab, and the developed task design and scripts are currently being extended upon by the related PhD student.


<span style="color:gray">December 2019 - May 2020</span><br>
**Research Intern**<br>
*Cognition Lab, Indian Institute of Science, Bangalore, India*

* Worked with multiple PhD students to conduct experimental tasks and decode EEG signal data.
* Pioneered the use of web-based experiments to remotely conduct experiments during COVID-19.
* Worked on my undergraduate thesis titled "Selection in Visual Working Memory due to Exogenous Visuospatial Attention", by employing a 4-AFC task.


<span style="color:gray">June 2018 - August 2018</span><br>
**Software Developer, Intern**<br>
*CGI, Bangalore, India*

* Developed a spelling corrector for a ChatBoth platform using LSTM recurrent neural networks.
* Employed sequence-to-sequence and character-based encoder-decoder model.


## **Projects**

<span style="color:gray">March 2023 - Present</span><br>**[An Active Learning Approach to Predicting Potency of Small Molecules for Antibacterial Screening](../assets/project_reports/automation_proposal.pdf)**<br>
*Carnegie Mellon University, Pittsburgh, USA*

* Predicting potency of small molecules on Burkholderia cenocepacia using active machine learning techniques, using SMILES data.
* Implementation of active machine learning techniques like Active Deep Reinforcement Learning, A2 algorithm, Expected Model Change and Query-by-committee selection strategies.


<span style="color:gray">March 2023 - Present</span><br>**[A Comparative Study and Improvement of Models Predicting Transcription Start Sites](../assets/project_reports/bioinfo_proposal.pdf)**<br>
*Carnegie Mellon University, Pittsburgh, USA*

* Comparison and improvement of latest TSS prediction pipelines, for better annotation of transcripts.
* Implementation of techniques like ADAPT-CAGE, DeepTSS and DeeReCT-TSS and integrating multiple sequence data types like CAGE-seq, RNA-seq, ChIP-seq, and DNase-seq for each model.
* Usage of STAR, HISTAT2 and MACS during pre-procesing of RNA-seq and CAGE-seq data.


<span style="color:gray">September 2022 - November 2022</span><br>
**[A single cell RNA-seq based aging clock for human neurons](../assets/project_reports/predict_age_report.pdf)**<br>
*Carnegie Mellon University, Pittsburgh, USA*

* Identified latent representations for efficient prediction of patient age.
* Developed Poisson Variational Autoencoder for efficient age prediction, which beat the baseline Multilayer Perceptron model and Standard VAE model performance with around 36% improvement.


<span style="color:gray">October 2022 - November 2022</span><br>
**[Common Signatures between Severe Asthma and Lung Cancer](../assets/project_reports/signatures_report.pdf)**<br>
*Carnegie Mellon University, Pittsburgh, USA*

* Implemented Gene Set Enrichment Analysis on data from GEO, to identify common signatures.
* Validated results by classifying a separate lung cancer dataset using Decision Trees.

<span style="color:gray">September 2022 - November 2022</span><br>
**[Simulating sequence evolution for in-silico experimentation](../assets/project_reports/simulations_report.pdf)**<br>
*Carnegie Mellon University, Pittsburgh, USA*

* Developed a simulation of bacterial transcription, translation, fitness, selection and mutation.
* This is a team project with five members, each responsible for one of the five steps mentioned above.
* Responsible for the mutation step, where a mutation is programmatically induced at random positions, sampled from a uniform or binomial distribution.


<span style="color:gray">February 2022 - March 2022</span><br>
**[Classification of Glioma](../assets/project_reports/glioma.pdf)**<br>
*Carnegie Mellon University, Pittsburgh, USA*

* Processed gene expression data from TCGA to classify glioma subtypes.
* Employed Gaussian Naive Bayes and Support Vector Machine to model the data
* Validated the results with a 5-fold cross-validation and achieved over 85% accuracy.


<span style="color:gray">October 2021 - November 2021</span><br>
**Natural Selection Simulator**<br>
*Carnegie Mellon University, Pittsburgh, USA*

* Developed a graphical game-like simulator to observe selection over several generations.

<span style="color:gray">March 2017 - April 2017</span><br>
**Robot arm using Arduino**<br>
*PES University, Bangalore, India*

* Equipped the robot arm with a camera, which was programmed to identify an object of a specific color and dimension with the help of OpenCV library. 
* Upon detection, the arm grabs the object.


<span style="color:gray">September 2016 - October 2016</span><br>
**Hit-and-score graphical game**<br>
*PES University, Bangalore, India*

* Designed the graphics of the game on Python using the Pygame library.
* Developed a shooting game that requires the player to shoot a moving object on the screen to increase the score.


## **Relevant Coursework**

**Carnegie Mellon University**

<br> 10617: Intermediate Deep Learning, 10725: Convex Optimization, 02613: Algorithms & Advanced Data Structures, 02620: Machine Learning, 02750: Automation Science, 03713: Bioinformatics Data Practicum

**PES University**

<br> Data Structures, Design and Analysis of Algorithms, Database Management Systems, Operating Systems, Molecular Modeling and Simulation



















